var/spack/repos/builtin/packages/py-dask-mpi/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyDaskMpi(PythonPackage):
+    """Deploying Dask using MPI4Py."""
+
+    homepage = "https://github.com/dask/dask-mpi"
+    url      = "https://pypi.io/packages/source/d/dask-mpi/dask-mpi-2.0.0.tar.gz"
+
+    import_modules = ['dask_mpi']
+
+    version('2.0.0', sha256='774cd2d69e5f7154e1fa133c22498062edd31507ffa2ea19f4ab4d8975c27bc3')
+
+    depends_on('py-dask@2.2:', type=('build', 'run'))
+    depends_on('py-mpi4py', type=('build', 'run'))
```

Comment:
no need of version for py-mpi4py ?

Reply:
ok no I checked https://github.com/dask/dask-mpi/blob/master/environment.yml ... 
this is strange tho.

Reply:
That environment file is just messed up. There's the `jupyter` reference in there that does not occur anywhere else in the code. Can't say I'm a fan of this way of specifying dependencies that seems to be interwoven with the testing requirements.